Truncated error message when failing to build virtual file system (VFS)

[//]: # (Thank you for taking the time to report an issue or propose an enhancement. Questions are also welcome.)
[//]: # (If you have more than one item to report, please open separate issues!)

[//]: # (This text will not be visible and you can freely delete it. Check the Preview tab for a cleaner look.)

### What is wrong
I'am not a programer but i know basics about compiling code on various operation systems.
In the moment i'am trying to make JA2 straciatella run on a Raspberry Pi 4 and a Raspi Zero W. 
I succesfully compiled the code with rustc version 1.48.0 and unziped the DATA file from the GOG Installer (German Version 1.12) to "/home/pi/Downloads/ja2"
I also setup the ja2.json file with the following content
            {
            // Put the directory to your original ja2 installation into the line below.
            "game_dir": "/home/pi/Downloads/ja2",
            "res": "1280x720",
            "resversion": "GERMAN",
            "fullscreen": false,
            "debug": true,
        }

When i try to launch the game it gives me an error message:

2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Initializing Memory Manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Initializing Game Resources
**2021-10-29T06:59:42 [ERROR] sgp/SGP.cc: Game has been terminated due to an unrecoverable error: Failed to build virtual file system (VFS): Vfs_init_from_engine_options EngineOptions { stracciatella_home: "/home/pi/.ja2", vanilla_game_dir: "/home/pi/Downloads/ja2", assets_dir: "/usr/local/bin",** 
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Shutting Down Button System
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Shutting Down Sound Manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Shutting Down Video Surface Manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/Video.cc: Shutting down the Video Surface manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Shutting Down Video Object Manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Shutting Down Video Manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/Video.cc: Shutting down the video manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Shutting Down Memory Manager
2021-10-29T06:59:42 [DEBUG] (1) sgp/SGP.cc: Shutting Down SDL
Abgebrochen

I think it has to do something with the DATA folder of JA2. Has anyone encountered the same error?

[//]: # (For bugs, please fill in this table [otherwise delete it]. The lines don't have to stay aligned.)

| JA2S version | Game version        | Vanilla bug? | Used to work? |
| ------------ | ------------------- | ------------ | ------------- |
|  0.18 | eg. JA2 GOG Deutsch (1.12)  |    yes    |    no     |

[//]: # (Please also include:)
[//]: # (- how to reproduce the bug and, if possible, a save game)
[//]: # (- the operating system)
[//]: # (- in case you know, for enhancements, if this feature was already added by some other source mod like 1.13)

[//]: # (When pasting backtraces or any other logs with hashed enumerations, please format it as a code block.)
[//]: # (You can do that by selecting it and clicking the "<>" icon or manually putting three backticks [```] at each end.)

--- a/COMPILATION.md
+++ b/COMPILATION.md
@@ -44,6 +44,9 @@ cmake path/to/source
 make
 ```
 
+If you want to be able to install the resulting binary on your system, please ensure that `CMAKE_INSTALL_PREFIX` matches
+with `EXTRA_DATA_DIR`. Example: `cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DEXTRA_DATA_DIR=/usr/local/share/ja2 path/to/source`
+
 ## Build for Windows on Linux using MinGW (cross build)
 
 Additional requirements: MinGW compiler

--- a/rust/stracciatella_c_api/src/c/vfs.rs
+++ b/rust/stracciatella_c_api/src/c/vfs.rs
@@ -39,10 +39,7 @@ pub extern "C" fn Vfs_init(
     let engine_options = unsafe_ref(engine_options);
     let mod_manager = unsafe_ref(mod_manager);
     if let Err(err) = vfs.init(engine_options, mod_manager) {
-        remember_rust_error(format!(
-            "Vfs_init_from_engine_options {:?}: {}",
-            engine_options, err
-        ));
+        remember_rust_error(format!("Vfs_init_from_engine_options: {}", err));
     }
     no_rust_error()
 }

--- a/src/sgp/SGP.cc
+++ b/src/sgp/SGP.cc
@@ -566,7 +566,7 @@ void TerminationHandler()
 		{
 		}
 	}
-	SLOGE(errorMessage.c_str());
+	STLOGE("{}", errorMessage);
 	#ifdef __ANDROID__
 	jniEnv->CallVoidMethod(exceptionContainerSingleton, setAndroidExceptionMethodId,
                                    jniEnv->NewStringUTF(errorMessage.c_str()));

--- a/src/sgp/SGP.cc
+++ b/src/sgp/SGP.cc
@@ -566,7 +566,7 @@ void TerminationHandler()
 		{
 		}
 	}
-	STLOGE("{}", errorMessage);
+	STLOGE(errorMessage.c_str());
 	#ifdef __ANDROID__
 	jniEnv->CallVoidMethod(exceptionContainerSingleton, setAndroidExceptionMethodId,
                                    jniEnv->NewStringUTF(errorMessage.c_str()));

